bundles/org.openhab.binding.matter/src/main/java/org/openhab/binding/matter/internal/actions/MatterOTBRActions.java

```diff
@@ -155,9 +157,11 @@ public void setThingHandler(@Nullable ThingHandler handler) {
         if (delay == null) {
             delay = 30000L;
         }
-        // default to generating a new pending timestamp
-        if (generatePendingTime == null || generatePendingTime.booleanValue()) {
-            tds.setPendingTimestamp(ThreadTimestamp.now(false));
+        if (Boolean.FALSE.equals(pushAsActive)) {
+            // default to generating a new pending timestamp
+            if (generatePendingTime == null || generatePendingTime.booleanValue()) {
```

Comment:
```suggestion
            if (!Boolean.FALSE.equals(generatePendingTime)) {
```

Reply:
yeah, its a good catch.  Its probably never null as is required, but yeah that was the original intent.